src/Webhook/Services/ChargeOrderService.php

```diff
@@ -239,9 +239,37 @@ protected function handleProcessing_TODO(Webhook $webhook)
 
     protected function handleAntifraudReproved(Webhook $webhook)
     {
+        $history = $this->addAntifraudHistoryComment('Antifraud reproved');
         return $this->handlePaymentFailed($webhook);
     }
 
+    protected function handleAntifraudApproved(Webhook $webhook)
+    {
+        $history = $this->addAntifraudHistoryComment('Antifraud aproved');
```

Comment:
Fica a seu critério, mas seria bom evitar esse boilerplate

Reply:
nao entendi, pode explicar melhor?

Reply:
Você tá jogando o retorno do método pra uma variável e logo em seguida retorna num array. Seria mais ou menos assim.
        return [
            "message" => $this->addAntifraudHistoryComment('Antifraud aproved');,
            "code" => 200
        ]; 

O código fica mais limpo, mas só é ruim pra debug, algumas pessoas criam a variável só pra facilitar nisso

Reply:
perfeito, vou fazer a mudança!